op-scim-bridge/chart/op-scim-bridge/templates/op-scim-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: op-scim-bridge
```

Comment:
There should be a variable here in the name, in case someone wants to deploy 2 bridges in the same cluster